pages/common/androguard.md

```diff
@@ -0,0 +1,16 @@
+# androguard
+
+> Python tool to play with Android files.
```

Comment:
I would prefer to state that it is a reverse engineering tool as "a tool to play" seem a little vague. 
Maybe something like
```suggestion
> Reverse engineering tool for Android applications. Written in Python.
```

What do you think?

Reply:
I reused the Debian description, but I agree yours is better :+1: